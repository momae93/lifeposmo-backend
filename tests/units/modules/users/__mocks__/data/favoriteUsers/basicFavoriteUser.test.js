const basicFavoriteUser = require('../../../../../../../src/modules/users/__mocks__/data/favoriteUsers/basicFavoriteUser');

describe('[MODULES][USERS][__MOCKS__][DATAS] : basic favorite user', () => {
  it('should match snapshot basic favorite user', async () => {
    expect(basicFavoriteUser).toMatchSnapshot();
  });
});
