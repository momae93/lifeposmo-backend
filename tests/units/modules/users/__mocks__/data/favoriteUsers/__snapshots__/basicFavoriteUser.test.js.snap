// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[MODULES][USERS][__MOCKS__][DATAS] : basic favorite user should match snapshot basic favorite user 1`] = `
Object {
  "idFavoriteUser": 1,
  "idUser": 2,
}
`;
